lib/utils/urls.ts

```diff
@@ -1,5 +1,5 @@
 export const siteUrl = (path: string = "") => {
-  let url = process.env.NEXT_PUBLIC_BASE_URL || "/";
+  let url = process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000/";
```

Comment:
is local host needed here?

Reply:
https://github.com/open-sauced/insights/blob/5ba312b8b3f31d48d60f983e87061c688ca81369/pages/user/%5Busername%5D/card.tsx#L125

I believe OG:image meta data properties need a full url including domain or else social media won't pick them up. This is the fallback when testing locally.

Reply:
We have a default `NEXT_PUBLIC_BASE_URL` in the `.env` so someone running this locally would specifically have to override it.